addons/mail/static/src/core/common/thread_model.js

```diff
@@ -354,7 +354,7 @@ export class Thread extends Record {
     }
 
     get displayToSelf() {
-        return this.is_pinned || (["channel", "group"].includes(this.type) && this.hasSelfAsMember);
+        return this.is_pinned || (this.hasMemberList && this.hasSelfAsMember);
```

Comment:
I would not use `hasMemberList` here, it's unrelated to `displayToSelf` feature, even if it might have the same condition inside.

Reply:
The condition is not the same actually, because hasMemberList is overriden in livechat to add it too, and we don't want that. Having a member in livechat should not override the pinned status.